client/components/status/status.js

```diff
@@ -108,6 +184,23 @@ export default class Status extends React.Component {
             </div>
           </div>
         </div>
+        {captivePortalForm && (window.top === window.self) &&
+          <>
+            <form ref={this.formRef} method={captivePortalForm.method || ''}
+              id="cp-form"
+              action={captivePortalForm.action || ''}
+              target="owisp-auth-iframe"
+              className="owisp-auth-hidden"
+            >
+              <input readOnly type="text" name={captivePortalForm.fields.username || ''} value={username} />
+              <input readOnly type="text" name={captivePortalForm.fields.password || ''} value={password} />
+              {captivePortalForm.additional_fields.length > 0 ? captivePortalForm.additional_fields.map(field =>
+                <input readOnly type="text" name={field.name} value={field.value} />
+              ) : null}
+            </form>
+            <iframe onLoad={this.handleIframe} ref={this.ifameRef} name="owisp-auth-iframe" className="owisp-auth-hidden" title='owisp-auth-iframe' />
```

Comment:
you're using single quotes for `owisp-auth-iframe` while double quotes for all other attrs